### PR TITLE
 Moderation(ban) links shown to a normal user are not required

### PR DESCRIPTION
**Pull request by ankitbhardwaj78**
#1972

Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

File: app/views/users/list.html.erb [hunk 1]
```diff
@@ -5,7 +5,7 @@
 			<p> This page shows a list of all Public Lab contributors and is sorted by those who have posted most recently. Click the profile link to view their posting activity. </p>
 		</div>
 	</div>
-  <% if current_user && current_user.role = "admin" && current_user.role = "moderator" %>
+  <% if current_user && current_user.role == "admin" && current_user.role == "moderator" %>
```
Comment (jywarren) [inline app/views/users/list.html.erb]: Also these should be OR, not And, for roles, right? Great catch!!

**Comment by ankitbhardwaj78** [comment]
@jywarren take a look i have made changes

**Comment by ankitbhardwaj78** [comment]
@jywarren this is complete and working fine for me..please take a review

**Comment by jywarren** [comment]
Perfect! 👌  🎉

**Pull request merged**